architecture/topics/aws_getting_started_before.adoc

```diff
@@ -0,0 +1,210 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-before-{context}']
+= Before you begin deploying the AWS Broker
+
+Before you can get started deploying the AWS Broker, make sure you meet the prerequisites described in this topic.
+
+.Prerequisites 
+
+The following items are required to deploy the AWS Broker into {product-title}.
+
+{product-title}::
+You must have an {product-title} cluster configured to run the Service Catalog.
+
+You must configure an {product-title} persistent volume (PV) for use by AWS Broker (1 GiB recommended).
+
+////
+AWS Service Broker:: 
```

Comment:
This is no longer accurate, the AWS Service Broker no longer talks to docker hub to discover services, it now connects to S3 and retrieves a list of services from there.

Reply:
Removed